core/src/main/java/bisq/core/grpc/ApiStatus.java

```diff
@@ -56,5 +53,5 @@ public String toString() {
                 ", description='" + description + '\'' +
                 '}';
     }
-}
+    }
```

Comment:
Whitespace. Doesn't matter as `ApiStatus` will be removed shortly enough, but it makes me wonder whether or not you're using Editorconfig as suggested at bisq-network/style#10?

Reply:
My Intellij IDE is using the project scheme defined in the repo's `.editorconfig` file:

```
root = true

[*]
indent_style = space
indent_size = 4
continuation_indent_size = 8
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true

[*.diff]
trim_trailing_whitespace = false

[Makefile]
indent_style = tab

[*.bat]
end_of_line = crlf

[build.gradle]
continuation_indent_size = 4

[.idea/codeStyles/*.xml]
indent_size = 2
insert_final_newline = false

```
It could have been a fat finger error when I was reformatting the class after removing some lines with ^ALT-L.  But I'm really not sure if that is the cause.  I'll keep an eye on it.